apps/api/v2/src/ee/bookings/controllers/bookings.controller.ts

```diff
@@ -366,9 +366,32 @@ export class BookingsController {
 
     if (err instanceof Error) {
       const error = err as Error;
+      if (errorCodes.includes(error.message)) {
+        throw new HttpException(error.message, 400);
+      }
       throw new InternalServerErrorException(error?.message ?? errMsg);
     }
 
     throw new InternalServerErrorException(errMsg);
   }
 }
+
```

Comment:
Should we not pull these from core so that if they update this code will not need to be changed and will automatically pick up any new error codes?

Reply:
yes we can, in this case it's not critical since handleNewBooking shouldn't be updated from platform libraries after the release of the refactor and this controller will be deprecated

Reply:
Ah I see. Ok.